### ISSUE TITLE
No text at all in a GUI (Zyn-Fusion)

### ISSUE THREAD

**Issue by sirius-c** [memononen/nanovg#549]
This is not necessarily a bug report, but rather to see if some further help could be provided.

In https://github.com/zynaddsubfx/zyn-fusion-issues/issues/151, I reported about a text display issue with Zyn-Fusion. Suggestion by the author of that program is that there might be an issue with the 'font atlas' in nanovg, which is used by that program.

Would there be some idea how to fix this problem?

**Comment by memononen** [comment]
NanoVG uses fontstash for font rendering, can you test the fontstash
example to see if you get similar problem:
https://github.com/memononen/fontstash

It uses premake too, so same procedure to run the example as with nanovg.

--mikko

**Comment by sirius-c** [comment]
Thanks for the suggestion, but the result when running the compiled `example` is identical to https://github.com/memononen/fontstash/blob/master/screenshots/screen-01.png?raw=true.

**Comment by memononen** [comment]
Yep, that's how it should look like. I was trying to see if we could
replicate the problem with simpler setup.

I have to say, I have no idea what might cause your problem. NanoVG demo
slowing down seems very odd. The error you get on console means that one of
the draw commands is doing "invalid operation". It can mean many things. It
could be buggy opengl driver, bug in nanovg, or zyn-fusion using the api
unexpected way. Sorry, I cant be more helpful this time.
